README.md

```diff
@@ -2,77 +2,80 @@
 
 # DDEV-CRON <!-- omit in toc -->
 
-- [Intro](#intro)
+- [Introduction](#introduction)
 - [Getting started](#getting-started)
 - [Implementation](#implementation)
 - [Examples](#examples)
+  - [TYPO3 scheduler](#typo3-scheduler)
+  - [Drupal cron](#drupal-cron)
+  - [Laravel cron](#laravel-cron)
 
-## Intro
+## Introduction
 
 This DDEV add-on helps to execute a command in the web container based on a cron schedule. Cron is a classic Linux/Unix service with a well-known configuration syntax.
 
-The add-on
+The add-on:
 
 - Installs and runs the cron service inside the web container
-- Adds a sample cron configuration that adds to a file every minute.
+- Adds an example job that writes out the current time.
 - Required DDEV v1.19.3 or higher.
 
-*This extension is designed to be a generic implentation. See [Running TYPO3 Cron inside the web container](https://github.com/ddev/ddev-contrib/tree/master/recipes/cronjob) for a specific example of a manual setup.*
+*This extension is designed to be a generic implementation. See [Running TYPO3 Cron inside the web container](https://github.com/ddev/ddev-contrib/tree/master/recipes/cronjob) for a specific example of a manual setup.*
 
 ## Getting started
 
-- Install the add-on with `ddev get ddev/ddev-cron`
-- Update the provided `.ddev/config.cron.yaml` as you see fit with your expected cron jobs (and remove the demonstration line). You can also just add those demonstration lines to your `.ddev/config.yaml` and delete the `.ddev/config.cron.yaml`.
-- `ddev restart`
+- Install the DDEV cron add-on:
 
-## Implementation
+  ```shell
+  ddev get ddev/ddev-browsersync
+  ```
 
-The provided `web-build/Dockerfile.ddev-cron` and `web-build/cron.conf` configure the traditional cron daemon to run inside the web container.
+- Update `./.ddev/web-build/custom.cron` with your required commands.
+  - Remove `#ddev-generated` to prevent DDEV from overriding the file.
```

Comment:
I think it might be better to try something that didn't require editing the #ddev-generated file, like maybe telling them to add another cron file that is *not* managed?

Reply:
This section has no longer exists.